css/styles.css

```diff
@@ -45,6 +45,71 @@ main {
     box-shadow: inset 0 0 9px 12px rgba(113,34,250,0.3), 0 0 9px 12px rgba(113,34,250,0.3);
 }
 
+aside div {
+    width: 70px;
+    height: 70px;
+    border: 8px solid white;
+    border-radius: 40px;
+    margin: 0 auto 15px auto;
+    background-size: cover;
+}
+
+.white {
+    border-color: #FFFFFF;
+    background-image: url("../IMG/brushWhite.png");
+    box-shadow: 0 0 5px 5px rgba(255,255,255, 0.3), inset 0 0 5px 5px rgba(255,255,255, 0.3);
+    -o-transition:.5s;
+    -ms-transition:.5s;
+    -moz-transition:.5s;
+    -webkit-transition:.5s;
+}
+
+.white:hover {
+    cursor: pointer;
+    box-shadow: 0 0 10px 10px rgba(255,255,255, 0.3), inset 0 0 5px 5px rgba(255,255,255, 0.6)
+}
+
+.white-active {
+    background-color: #FFFFFF;
+    background-image: url("../IMG/brushBlack.png");
+    border: 4px solid #FFFFFF;
+    box-shadow: 0 0 10px 10px rgba(255,255,255, 0.3);
+    -o-transition:.5s;
+    -ms-transition:.5s;
+    -moz-transition:.5s;
+    -webkit-transition:.5s;
+    cursor: pointer;
+}
+
+.eraser {
+    margin-top: 30px;
+    border-color: #b2b2b2;
+    background-image: url("../IMG/eraserGrey.png");
+    box-shadow: 0 0 5px 5px rgba(178,178,178, 0.3), inset 0 0 5px 5px rgba(178,178,178, 0.3);
+    -o-transition:.5s;
+    -ms-transition:.5s;
+    -moz-transition:.5s;
+    -webkit-transition:.5s;
+}
+
+.eraser:hover {
+    cursor: pointer;
+    box-shadow: 0 0 10px 10px rgba(178,178,178, 0.3), inset 0 0 5px 5px rgba(178,178,178, 0.6)
+}
+
+.eraser-active {
+    margin-top: 30px;
+    background-color: #b2b2b2;
+    background-image: url("../IMG/eraserBlack.png");
+    border: 4px solid #b2b2b2;
+    box-shadow: 0 0 10px 10px rgba(178, 178, 178, 0.3);
+    -o-transition: .5s;
+    -ms-transition: .5s;
+    -moz-transition: .5s;
+    -webkit-transition: .5s;
+    cursor: pointer;
```

Comment:
This is repeated a bunch of times, cant you just put it on main?

Reply:
done - cursor on the button class